index.html

```diff
@@ -13,8 +14,24 @@
         </div>
             
         <div class="menu">
-            Menu
+            <div>
+            <i class="fas fa-bars icon"></i>
+            </div>
+            <div>
+                Menu
+            </div>
+        </div>
+        <div class="slide-menu active">
+            <div class="close-menu">
+                <i class="fas fa-times"></i>
+            </div>
+            <ul class="list">
+                <li class="item">Home</li>
+                <li class="item">Elements</li>
+                <li class="item">Generic</li>
+            </ul>
         </div>
```

Comment:
Always separate containers with new line. That will increase code readability.
```suggestion
        <div class="slide-menu active">
            <div class="close-menu">
                <i class="fas fa-times"></i>
            </div>
            
            <ul class="list">
                <li class="item">Home</li>
                <li class="item">Elements</li>
                <li class="item">Generic</li>
            </ul>
        </div>
```

Reply:
@AhmedBHameed I fixed the issues, shall I squash and merge now?

Reply:
@Farah7Tawfeek 

You still have fonts folder. After you remove it, you are ready to `squash and merge`.